lib/include/irritator/examples.hpp

```diff
@@ -320,6 +320,260 @@ example_qss_negative_lif(simulation& sim, F f) noexcept
     return status::success;
 }
 
+/** @brief Implements the Linear SEIR model for QSS1, QSS2 and QSS3.
+     *
+     * @details
+     *
+     * @Tparam QssLevel
+     * @Tparam F
+     * @param sim
+     * @param f
+     * @return
+     */
+
+    template<int QssLevel, typename F>
+    status
+        example_qss_seir_lineaire(simulation& sim, F f) noexcept
+    {
+        static_assert(1 <= QssLevel && QssLevel <= 3, "Only for Qss1, 2 and 3");
+
+        bool success = sim.can_alloc<abstract_wsum<QssLevel, 2>>(2) &&                                    
+            sim.can_alloc<abstract_multiplier<QssLevel>>(2) &&
+            sim.can_alloc<abstract_integrator<QssLevel>>(4) &&
+            sim.can_alloc<constant>(2) &&
+            sim.can_connect(12);//pour le seir_linear qu'est-ce que le nombre de connection 12,
```

Comment:
The function `sim.can_connect(12)` checks if the `irt::simulation` object can allocate `12` new connections between models. In irritator simulation kernel (`irritator/core.hpp`), all objects are pre-allocated. No memory allocation is allowed during simulation.